include/mapbox/geojsonvt/clip.hpp

```diff
@@ -115,6 +115,7 @@ class clipper {
             const auto& b = line[i + 1];
             const double ak = get<I>(a);
             const double bk = get<I>(b);
+            const bool lastSegFlag = (i == (len - 2));
```

Comment:
Nit: I don't like the word "flag" in variables, maybe `isLastSeg` would sound nicer?